packages/image-utils/src/sharp.ts

```diff
@@ -12,7 +13,7 @@ export async function resizeBufferAsync(buffer: Buffer, sizes: number[]): Promis
   // Create buffer for each size
   const resizedBuffers = await Promise.all(
     sizes.map(dimension => {
-      const density = (dimension / Math.max(metadata.width, metadata.height)) * metadata.density;
+      const density = dimension / Math.max(metadata.width, metadata.height) * metadata.density;
```

Comment:
"I don't take responsibility at all" for this Prettier change, but I can revert it if required 😄

Reply:
we need to fix our husky config so it stops doing this.

Reply:
partially fixed https://github.com/expo/expo-cli/issues/1920

Reply:
Thanks, just rebased and removed these 2 changes